plasmapy/physics/transport.py

```diff
@@ -129,7 +129,8 @@ def Coulomb_logarithm(T,
     logarithm to the form:
 
     .. math::
-        \ln{\Lambda} \equiv 0.5 \ln\left(1 + \frac{b_{max}^2}{b_{min}^2} \right)
+        \ln{\Lambda} \equiv 0.5 \ln\left(1 +
+        \frac{b_{max}^2}{b_{min}^2} \right)
```

Comment:
This might actually break docs rendering the latex formula... It's reasonable to keep these in one line despite pep8 shouting at us.

Reply:
Agreed, reverting these two changes.